import-service/src/handlers/importFileParser/handler.ts

```diff
@@ -0,0 +1,67 @@
+import { S3Event, S3EventRecord } from "aws-lambda";
+import { S3Client, GetObjectCommand, CopyObjectCommand, DeleteObjectCommand } from "@aws-sdk/client-s3";
+import csv from "csv-parser";
+import { Stream, Writable } from "stream";
+import util from "util";
+
+const pipelineAsync = util.promisify(Stream.pipeline);
```

Comment:
As far as I see in Node 20 (LTS) pipeline return promise natively:

https://nodejs.org/docs/latest-v20.x/api/stream.html#streams-promises-api

Reply:
done